wordpress-mobile/WordPress-iOS
# [Gutenberg] - PR 01 - Move gutenberg switch to site settings

This PR moves the Editor selector (Gutenberg switch) from App Settings to Site Settings.

Even though it makes sense to add it as part of the WRITING section, I decided to make it its own section to be able to keep displaying the footer. (cc @iamthomasbishop )
![editor_setting](https://user-images.githubusercontent.com/9772967/61517390-6b1bfc80-aa07-11e9-9d6c-de45f4289a44.png)


This is a UI only change, and the setting is still stored globally.
On a next PR, we will make the setting per-site.

To test:
- Check that the Switch displays correctly on the Site Settings screen.
- Check that the switch behaves correctly in the same way than it did before.

Update release notes:
(Will update release notes when the whole feature is ready)
- [*] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
